Modelica/Electrical/Analog/Interfaces.mo

```diff
@@ -125,43 +125,17 @@ The reason could be that
 
   partial model TwoPin "Component with two electrical pins"
     SI.Voltage v "Voltage drop of the two pins (= p.v - n.v)";
-    PositivePin p "Positive electrical pin" annotation (Placement(
-          transformation(extent={{-110,-10},{-90,10}})));
-    NegativePin n "Negative electrical pin" annotation (Placement(transformation(extent={{
-              90,-10},{110,10}})));
```

Comment:
This must be an accident. Removing the pins shouldn't be done.

Reply:
Yes, this happened accidentally